xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1540 @@
+      <Implements>
+        <InterfaceMember>P:System.ServiceModel.ICommunicationObject.State</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.ServiceModel.CommunicationState</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the current state of the communication-oriented object.</summary>
+        <value>The value of the <see cref="T:System.ServiceModel.CommunicationState" /> of the object.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.ICommunicationObject&gt; interface.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.IContextChannel.AllowOutputBatching">
+      <MemberSignature Language="C#" Value="bool System.ServiceModel.IContextChannel.AllowOutputBatching { get; set; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance bool System.ServiceModel.IContextChannel.AllowOutputBatching" />
+      <MemberSignature Language="DocId" Value="P:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#IContextChannel#AllowOutputBatching" />
+      <MemberSignature Language="VB.NET" Value=" Property AllowOutputBatching As Boolean Implements IContextChannel.AllowOutputBatching" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.IContextChannel.AllowOutputBatching : bool with get, set" Usage="System.ServiceModel.IContextChannel.AllowOutputBatching" />
+      <MemberSignature Language="C++ CLI" Value="property bool System::ServiceModel::IContextChannel::AllowOutputBatching { bool get(); void set(bool value); };" />
+      <MemberType>Property</MemberType>
+      <Implements>
+        <InterfaceMember>P:System.ServiceModel.IContextChannel.AllowOutputBatching</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Boolean</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets or sets a value that instructs Windows Communication Foundation (WCF) to store a set of messages before giving the messages to the transport.</summary>
+        <value>
+          <see langword="true" /> if the batching of outgoing messages is allowed; otherwise, <see langword="false" />.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.IContextChannel&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Channels.ServiceChannelProxy"/> instance is cast to an <see cref="T:System.ServiceModel.IContextChannel"/> interface.</remarks>

```